lib/wallaby.ex

```diff
@@ -104,4 +111,17 @@ defmodule Wallaby do
   defp default_driver do
     Application.get_env(:wallaby, :driver, Wallaby.Phantom)
   end
+
+  def driver do
```

Comment:
It replaces the default_driver call. We need it to work out which driver the user really wants to use.